src/test/java/lithium/university/TwitterPublishTest.java

```diff
@@ -0,0 +1,71 @@
+package lithium.university;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+import twitter4j.Status;
+import twitter4j.Twitter;
+import twitter4j.TwitterException;
+
+
+@RunWith(MockitoJUnitRunner.class)
+public class TwitterPublishTest {
+    @Mock
+    private Twitter twitterTest;
+
+    @InjectMocks
+    private TwitterPublish twitterPublishTest;
+
+    @Before
+    public void init() throws TwitterException {
+        /*General status message for twitterPublish behavior*/
+        Status s = Mockito.mock(Status.class);
+        Mockito.when(s.getText()).thenReturn("General status message for testing.");
+        Mockito.when(twitterTest.updateStatus(Mockito.anyString())).thenReturn(s);
+    }
+
+    private String genStringLength(int charLength) {
+        String ret = "";
+        for (int i = 0; i < charLength; i++) {
+            ret += "!";
+        }
+        return ret;
+    }
+
+    @Test
+    public void testPublishCharLengthZero() throws TwitterException {
+        boolean publishTest = true;
+        publishTest = twitterPublishTest.postToTwitter(twitterTest, genStringLength(0), TwitterApplication.TWEET_LENGTH);
+        Assert.assertEquals(false, publishTest);
+        System.out.println("testPublishCharLengthZero pass");
```

Comment:
you don't actually need the System.out.println statements.  The asserts should take care of the checking.